.releaserc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "branches": [
+    "+([0-9])?(.{+([0-9]),x}).x",
+    "main",
+    "next",
+    "next-major",
+    { "name": "beta", "prerelease": true },
+    { "name": "alpha", "prerelease": true }
+  ],
+  "plugins": ["@semantic-release/commit-analyzer"]
```

Comment:
do these need to be installed in package.json

Reply:
I figured that they don't as they are not in the [package.json](https://github.com/seamapi/makenew-tsmodule/blob/2887311c5d6d1b660dc281a57c526c173e88de12/package.json#L60) on makenew-tsmodule